source/includes/_examples.md

```diff
@@ -0,0 +1,219 @@
+  "fhirServer": "http://fhirtest.uhn.ca/baseDstu2",
+  "hook": "medication-prescribe",
+  "user": "Practitioner/example",
+  "context": [
+    {
+      "resourceType": "MedicationOrder",
+      "id": "medrx001",
+      "dateWritten": "2017-05-05",
+      "status": "draft",
+      "patient": {
+        "reference": "Patient/example"
+      },
+      "medicationCodeableConcept": {
+        "coding": [
+          {
+            "system": "http://www.nlm.nih.gov/research/umls/rxnorm",
+            "code": "857001",
+            "display": "Acetaminophen 325 MG / Hydrocodone Bitartrate 10 MG Oral Tablet"
+          }
+        ]
+      },
+      "dosageInstruction": [
+        {
+          "text": "Take 1 tablet Oral every 4 hours as needed",
+          "timing": {
+            "repeat": {
+              "frequency": 6,
+              "frequencyMax": 6,
+              "period": 1,
+              "unit": "d"
+            }
+          },
+          "asNeededBoolean": true,
+          "doseQuantity": {
+            "value": 10,
+            "unit": "mg",
+            "system": "http://unitsofmeasure.org",
+            "code": "mg"
+          }
+        }
+      ]
+    }
+  ],
+  "patient": "Patient/Patient-12214",
+  "prefetch": {
+    "medication": {
+      "response": {
+        "status": "200 OK"
+      },
+      "resource": {
+        "resourceType": "MedicationOrder",
+        "id": "medrx002",
+        "dateWritten": "2017-04-25",
+        "status": "active",
+        "patient": {
+          "reference": "Patient/Patient-12214"
+        },
+        "medicationCodeableConcept": {
+          "coding": [
+            {
+              "system": "http://www.nlm.nih.gov/research/umls/rxnorm",
+              "code": "1049621",
+              "display": "Oxycodone Hydrochloride 5 MG Oral Tablet"
+            }
+          ]
+        },
+        "dosageInstruction": [
+          {
+            "text": "Take 1 tablet by mouth every 4 hours as needed for pain.",
+            "timing": {
+              "repeat": {
+                "frequency": 6,
+                "period": 1,
+                "periodUnits": "d"
+              }
+            },
+            "asNeededBoolean": true,
+            "doseQuantity": {
+              "value": 5,
+              "unit": "mg",
+              "system": "http://unitsofmeasure.org",
+              "code": "mg"
+            }
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+This example illustrates the use of the CDS Hooks `medication-prescribe` hook to implement Recommendation #5 from the [CDC guideline for prescribing opioids for chronic pain](https://guidelines.gov/summaries/summary/50153/cdc-guideline-for-prescribing-opioids-for-chronic-pain---united-states-2016#420).
+
+This example is taken from the [Opioid Prescribing Support Implementation Guide](http://build.fhir.org/ig/cqframework/opioid-cds/), developed in partnership with the Centers for Disease Control and Prevention [(CDC)](https://www.cdc.gov/).
```

Comment:
Note: We'll need to change this URL off of the build domain to when we get the final published IG